### DISCUSSION TITLE
Welcome to ML-Fusion-Lab Discussions!

### DISCUSSION THREAD

**Discussion by vivekv2810** [ML-Fusion-Lab/ML-Fusion-Lab-Website#1] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Khushi14404** [comment]
@vivekv2810 Please add label to the issue

**Reply by vivekvardhan2810** [reply]
Hi, @Khushi14404, as this issue was presented by other user, so it will be first on first serve, if that user doesn't do this work

Then I will assign you the issue.

But still you can look out for other issues.
